csharp-package/brainflow/brainflow/board_controller_library.cs

```diff
@@ -63,7 +63,9 @@ public enum BoardIds
         NOTION_2_BOARD = 14,
         IRONBCI_BOARD = 15,
         GFORCE_PRO_BOARD = 16,
-        FREEEEG32_BOARD = 17
+        FREEEEG32_BOARD = 17,
+        MUSE_1_BOARD = 18,
```

Comment:
I think there is 0 chance that support for both muse devices will be added in single PR, more likely it will be splitted by multiple PRs

Reply:
sounds like it's better to leave the board_id out if there's no work to implement it yet